bootlib.ks

```diff
@@ -1,20 +1,15 @@
-local home is "0:/n/"+ship:name+"/".
-local pack is "0:/p/".
-local package is lex().
+local _ is lex("home", "0:/n/"+ship:name+"/", "pack", "0:/p/",
+"st", {parameter f. if homeconnection:isconnected and exists(f) copypath(f,_:home+f).},
+"ev", {parameter m. hudtext(m,5,2,24,WHITE,true).}).
```

Comment:
providing import("boot"):ev(m) to display a message on the screen
and on the terminal.